src/test/scala/org/broadinstitute/hail/variant/vsm/VSMSuite.scala

```diff
@@ -306,7 +306,7 @@ class VSMSuite extends SparkSuite {
 
   @Test def testCoalesce() {
     val g = for (
-      vsm <- VariantSampleMatrix.gen[Genotype](sc, VSMSubgen.random);
+      vsm <- VariantSampleMatrix.gen[Genotype](sc, VSMSubgen.random).filter(_.nPartitions > 0);
```

Comment:
We should be able to call `coalesce` on RDDs with empty partitions.  This needs to get fixed.